Mathlib/Combinatorics/Additive/Energy.lean

```diff
@@ -45,6 +41,17 @@ def multiplicativeEnergy (s t : Finset α) : ℕ :=
 #align finset.multiplicative_energy Finset.multiplicativeEnergy
 #align finset.additive_energy Finset.additiveEnergy
 
+@[inherit_doc multiplicativeEnergy] notation3:max "Eₘ[" s ", " t "]" => multiplicativeEnergy s t
+@[inherit_doc additiveEnergy] notation3:max "E[" s ", " t "]" => additiveEnergy s t
```

Comment:
Could you scope these notations and the next ones? They are a little bit too generic to be always active. Also, maybe write `Finset.multiplicativeEnergy s t` to avoid issues when `Finset` is not open.